examples/configuration_api_example.py

```diff
@@ -0,0 +1,26 @@
+''' Configuration API client example usage. '''
+
+from livechat.configuration.client import ConfigurationApi
+
+# Get group name.
+configuration_api = ConfigurationApi.get_client(token='<your access token>')
+groups = configuration_api.list_groups()
+group_1 = groups.json()[1]['id']
```

Comment:
In my opinion, `group_1 = groups.json()[1].get('id')` is more Pythonic ;) (applicable to the similar occurrences)

Reply:
👍